[2024-04-18T08:26:56.271+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-18T08:26:56.390+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-04-15T00:00:00+00:00 [queued]>
[2024-04-18T08:26:56.428+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-04-15T00:00:00+00:00 [queued]>
[2024-04-18T08:26:56.433+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-18T08:26:56.535+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-15 00:00:00+00:00
[2024-04-18T08:26:56.630+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'scheduled__2024-04-15T00:00:00+00:00', '--job-id', '170', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpny108s4i']
[2024-04-18T08:26:56.632+0000] {standard_task_runner.py:91} INFO - Job 170: Subtask get_files
[2024-04-18T08:26:56.652+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=159) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-18T08:26:56.656+0000] {standard_task_runner.py:63} INFO - Started process 234 to run task
[2024-04-18T08:26:56.795+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.get_files scheduled__2024-04-15T00:00:00+00:00 [running]> on host 2ce858e27684
[2024-04-18T08:26:57.401+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-15T00:00:00+00:00'
[2024-04-18T08:26:57.410+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-18T08:26:57.411+0000] {logging_mixin.py:188} INFO - 1 files found in /opt/***/dags/data
[2024-04-18T08:26:57.411+0000] {python.py:237} INFO - Done. Returned value was: ['/opt/***/dags/data/order_bazilian.csv']
[2024-04-18T08:26:57.411+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-18T08:26:57.538+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240415T000000, start_date=20240418T082656, end_date=20240418T082657
[2024-04-18T08:26:57.694+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-18T08:26:57.860+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-18T08:26:57.869+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
